app/views/docs/realtime.phtml

```diff
@@ -52,10 +58,16 @@ client
 ;
 
 final realtime = Realtime(client);
-final subscription = realtime.subscribe(channel);
 
-subscription.stream.listen(callback);
-</code></pre>
+// Subscribe to files channel
+final subscription = realtime.subscribe(['files']);
+
+subscription.stream.listen((response) {
+    if(response.event == 'storage.files.create') {
```

Comment:
```suggestion
    if(response.event === 'storage.files.create') {
```

No idea if this is supported. Let's either try it, or ask Damodar/Jake

Reply:
I believe it's actually supposed to be `==`

Change from previous conversation: https://github.com/appwrite/docs/pull/182#pullrequestreview-965499823